velox/connectors/hive/iceberg/PositionalDeleteFileReader.cpp

```diff
@@ -247,18 +247,6 @@ void PositionalDeleteFileReader::updateDeleteBitmap(
         deletePositions[deletePositionsOffset_] - rowNumberLowerBound);
     deletePositionsOffset_++;
   }
-
-  // There might be multiple delete files for a single base file. The size of
-  // the deleteBitmapBuffer should be the largest position among all delte files
-  deleteBitmapBuffer->setSize(std::max(
```

Comment:
If you remove this, where do we set the size of the delete bitmap? Suppose there is no delete positions matched in the current range, the size of the bit map should be zero. If you don't set it the bitmap would be passed to the reader with the size of nbytes(actualSize) and may regress the performance